datasampling.sh

```diff
@@ -0,0 +1,57 @@
+package: DataSampling
+version: "%(tag_basename)s"
+tag: v0.1.0
+requires:
+  - boost
+  - "GCC-Toolchain:(?!osx)"
+  - Common-O2
+  - InfoLogger
+  - FairRoot
+build_requires:
+  - CMake
+source: https://github.com/AliceO2Group/DataSampling
+incremental_recipe: |
+  make ${JOBS:+-j$JOBS} install
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+#!/bin/bash -ex
+
+case $ARCHITECTURE in
+  osx*) BOOST_ROOT=$(brew --prefix boost) ;;
+esac
+
+cmake $SOURCEDIR                                              \
+      -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                     \
+      ${BOOST_VERSION:+-DBOOST_ROOT=$BOOST_ROOT}              \
+      ${COMMON_O2_VERSION:+-DCommon_ROOT=$COMMON_O2_ROOT}     \
+      ${INFOLOGGER_VERSION:+-DInfoLogger_ROOT=$INFOLOGGER_ROOT} \
+      ${FAIRROOT_VERSION:+-DFAIRROOTPATH=$FAIRROOT_ROOT}      \
+      ${FAIRROOT_VERSION:+-DFairRoot_DIR=$FAIRROOT_ROOT}                              
+
+make ${JOBS+-j $JOBS} install
+
+#ModuleFile
+mkdir -p etc/modulefiles
+cat > etc/modulefiles/$PKGNAME <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0                                                          \
```

Comment:
I have had troubles with double backslash in the case a variable such as GCC_TOOLCHAIN_VERSION did not exist. Thus using a single backslash which will always work.